tenderly-suite/flow-limit-update/flow-limit-update/flowLimitUpdate.js

```diff
@@ -0,0 +1,89 @@
+const axios = require('axios').default;
+const { ethers } = require('ethers');
+
+const PAGER_DUTY_ALERT_URL = 'https://events.pagerduty.com/v2/enqueue';
+const Severity = {
+    INFO: 'info',
+    CRITICAL: 'critical',
+    WARNING: 'warning',
+    1: 'info',
+    2: 'warning',
+};
+
+const flowLimitUpdateFn = async (context, event) => {
+    if (!event || !event.logs || !context || !context.metadata) {
+        throw new Error('INVALID_INPUT_FOR_ACTION');
+    }
+
+    const { flowLimitSet } = await context.storage.getJson('EventsABI');
+    const flowLimitSetHash = ethers.utils.keccak256(ethers.utils.toUtf8Bytes(flowLimitSet));
+
+    const itsAddresses = await context.storage.getJson('ITSAddresses');
+
+    let severity = 0;
+    const tokenIDs = [];
+    const tokenManagers = [];
+    const operators = [];
+
+    for (const log of event.logs) {
+        if (log.topics[0] === flowLimitSetHash) {
+            console.log(`event emitted: ${flowLimitSet}`);
+
+            if (log.topics.length < 2) {
```

Comment:
```suggestion
            if (log.topics.length !== 2) {
```
If you expect the topics length to be exactly 2